python_modules/dagster/dagster/_core/pipes/subprocess.py

```diff
@@ -101,9 +102,10 @@ def run(
                 command,
                 cwd=cwd or self.cwd,
                 env={
-                    **pipes_session.get_bootstrap_env_vars(),
+                    **os.environ,
```

Comment:
if we get user demand we can pretty easily add a bool argument to disable this if we want